docs/install/2-install-server-with-podman-and-systemd.rst

```diff
@@ -0,0 +1,341 @@
+
+    .. only:: iso
+
+        .. code-block:: sh
+            :substitutions:
+
+            mkdir -p ~/.config/inmanta
+            podman run --rm -ti containers.inmanta.com/containers/service-orchestrator:|version_major| cat /etc/inmanta/inmanta.cfg > ~/.config/inmanta/inmanta.cfg
+
+2.  Update database settings:
+    It is very unlikely that your database setup will match the one described in the default config we just got.  Update the configuration in the ``[database]`` section
+    to reflect the setup you have.
+
+3.  Make sure that there is a folder on your host that can persist all the logs of the server and that it is owned by the user running the orchestrator service.  
+    In this setup, the folder on the host will be ``~/.local/share/inmanta-orchestrator-server/logs``.
+
+    .. code-block:: sh
+
+        mkdir -p ~/.local/share/inmanta-orchestrator-server/logs
+
+    .. warning:: 
+        Inside of the container, this folder will be mounted at ``/var/log/inmanta`` as it is the default location where the orchestrator saves its logs.  This
+        location is configurable in the orchestrator configuration file.  If you were, for any reason, to change this location, make sure to update any usage
+        of the ``/var/log/inmanta`` folder in the next installation steps.
+
+.. only:: iso
+
+    4.  Get the license files:
+        Together with the access to the inmanta container repo, you should also have received a license and an entitlement file. The orchestrator will need them
+        in order to run properly.  You can also place them in a config directory on your host.  After this step, we assume that this folder is
+        ``~/.config/inmanta/license/`` and that both files are named ``com.inmanta.license`` and ``com.inmanta.jwe`` respectively.
+
+        .. code-block:: console
+
+            $ tree .config/inmanta
+            .config/inmanta
+            ├── inmanta.cfg
+            └── license
+                ├── com.inmanta.jwe
+                └── com.inmanta.license
+
+            2 directories, 3 files
+
+
+.. _setup-systemd-unit:
+
+Start the server with systemd
+#############################
+
+Here is a systemd unit file that can be used to deploy the server on your machine.
+
+.. only:: oss
+
+    .. code-block:: 
+
+        [Unit]
+        Description=Podman 
+        Documentation=https://docs.inmanta.com
+        Wants=network-online.target
+        After=network-online.target
+        RequiresMountsFor=%t/containers
+
+        [Service]
+        Environment=PODMAN_SYSTEMD_UNIT=%n
+        Restart=on-failure
+        TimeoutStopSec=70
+        ExecStart=/usr/bin/podman run \
+                --cidfile=%t/%n.ctr-id \
+                --cgroups=no-conmon \
+                --sdnotify=conmon \
+                -d \
+                --replace \
+                --publish=127.0.0.1:8888:8888 \
+                --uidmap=993:0:1 \
```

Comment:
I would like a note that explains what these numbers are. Something like:

993 is the uid and gid of the inmanta user and group inside the container. This option instructs podman to map this user to our local non-root user.

Reply:
I added an explanation, is it clear enough or maybe too detailed?